Lib/pathlib.py

```diff
@@ -707,6 +665,62 @@ def match(self, path_pattern, *, case_sensitive=None):
                 return False
         return True
 
+
+class PurePath(_BasePurePath):
+    """Base class for manipulating paths without I/O.
+
+    PurePath represents a filesystem path and offers operations which
+    don't imply any actual filesystem I/O.  Depending on your system,
+    instantiating a PurePath will return either a PurePosixPath or a
+    PureWindowsPath object.  You can also instantiate either of these classes
+    directly, regardless of your system.
+
+    On Posix, calling ``bytes()`` on a path gives the raw filesystem path as
```

Comment:
What do you think of:
```suggestion
    On Posix, calling ``bytes(path)`` gives the raw filesystem path as
```